.github/workflows/tox.yml

```diff
@@ -86,31 +86,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Setup Python ${{ matrix.python-version }}
+      - name: Setup Python
         uses: actions/setup-python@v2
         with:
           python-version: 3.7
       - name: Install Python dependencies
         run: |
-          python -m pip install -r requirements/project.txt
```

Comment:
doing this to be faster I guess?

Reply:
Actually I'm more closely following deploy-release.yml.  This way we can be more certain that that workflow won't fail because this will continuously check it in a very similar way

Reply:
Ok, thanks